### PR TITLE
Integrate 8/11 Nightly RN Build

### PR DESCRIPTION
**Pull request by NickGerleman**
Commits: https://github.com/facebook/react-native/compare/f592ad050...cbec66ef9

All changes were mechanical

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/8473)

### CODE REVIEW

**Review by dannyvv — APPROVED** [review]

File: vnext/src/Libraries/Text/Text.windows.js [hunk 1]
```diff
@@ -150,15 +153,40 @@ const Text: React.AbstractComponent<
...
+  if (numberOfLines != null && !(numberOfLines >= 0)) {
+    console.error(
+      `'numberOfLines' in <Text> must be a non-negative number, received: ${numberOfLines}. The value will be set to 0.`,
+    );
+    numberOfLines = 0;
+  }
+
   const hasTextAncestor = useContext(TextAncestor);
 
+<<<<<<< Upstream
```
Comment (dannyvv) [inline vnext/src/Libraries/Text/Text.windows.js:166]: Looks like some merge conflicts are left here...

Comment (NickGerleman) [inline vnext/src/Libraries/Text/Text.windows.js:166]: whoops...

**Review by dannyvv — APPROVED** [review]

File: vnext/src/Libraries/Text/Text.windows.js (hunk 1 above)
Comment (NickGerleman) [reply vnext/src/Libraries/Text/Text.windows.js:166]: Yeah, just noticed 😂

**Comment by NickGerleman** [comment]
Need to take a closer look at Fabric PlatformColor changes

**Pull request merged**